src/app/search/items/search-filters/known-values.ts

```diff
@@ -228,6 +229,25 @@ const knownValuesFilters: ItemFilterDefinition[] = [
       return (item) => breakerType.includes(item.breakerType?.hash as BreakerTypeHashes);
     },
   },
+  {
+    keywords: 'breakermod',
+    description: tl('Filter.BreakerMod'),
+    format: 'query',
+    suggestions: Object.keys(breakerTypes),
+    destinyVersion: 2,
+    filter: ({ filterValue }) => {
+      const breakerType: BreakerTypeHashes[] | undefined =
+        breakerTypes[filterValue as keyof typeof breakerTypes];
+      if (!breakerType) {
+        throw new Error(`Unknown breaker type ${breakerType as string}`);
+      }
+      const breakingIchs = breakerType.flatMap((ty) => artifactBreakerMods[ty] || []);
+      return (item) =>
+        Boolean(
+          !item.breakerType && item.itemCategoryHashes.some((ich) => breakingIchs.includes(ich)),
```

Comment:
Should this really be a new search, or should the `breaker` search just take into account unlocked artifact mods? What's the use case for finding items that are *only* granted breaker by the artifact?

And then we'd probably want to think about whether to show the breaker icon on the item tooltip.

Reply:
> Should this really be a new search, or should the breaker search just take into account unlocked artifact mods?

I'm in favor of combining the logic into `breaker`, provided it handles cases like "Wish-Ender has intrinsic anti-barrier therefore it cannot get anti-overload from Overload Bow." I can't think of a scenario where I'd want to filter-search for whether it was specifically bound to the artifact mod. 

That said, it may be valuable to expose the breaker-"reason" as something that can be spot-checked on a particular item, such as icon/tooltip indicating "Le Monarque: Anti-Overload from permanent intrinsic" or "Hush: Anti-Overload from Seasonal Mod."

Reply:
In that case, would you expect the breaker icon to also display on the item tile if an artifact mod could grant it breaker?

Reply:
> would you expect the breaker icon to also display on the item tile if an artifact mod could grant it breaker?

Yeah, as someone who plays Too Damn Much, all the exotic weapons with breaker-status have permanent names/effects, distinct icons, and tend to get memorized and I don't even look for any icons. So showing seasonally-conditional results would be more helpful.